k8s/vault.yaml

```diff
@@ -1,18 +1,32 @@
+apiVersion: extensions/v1beta1
```

Comment:
Dumb question.  Are we still on beta on the Ingress API?

Reply:
Followed https://cloud.google.com/kubernetes-engine/docs/tutorials/http-balancer#step_3_create_an_ingress_resource for this component.